postman/schemas/transfer.yml

```diff
@@ -593,6 +593,212 @@ paths:
             application/json:
               schema:
                 "$ref": "#/components/schemas/response401"
+  "/quotes":
+    get:
+      tags:
+      - Helpers
+      summary: Get Quotes
+      description: |
+        Given a corridor and a sender amount, this operation returns
+        quotes which include the exchange rate, fee adjustments, disclosures, and
```

Comment:
As a consumer, I might be asking myself "Why multiple quotes? What's the difference between them?" We might want to explain that here or link back to a section in the guides that explains how and why we offer the user multiple quotes.

Reply:
Emilie to write something up and run it through product.

Reply:
Added a new sentence to this area